src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -432,6 +432,14 @@ Sets which network IP addresses the dev server and preview server should listen
 Do not use the `--host` flag to expose the dev server and preview server in a production environment. The servers are designed for local use while developing your site only.
 :::
 
+### `--allowed-hosts`
+
+<p><Since v="5.4.0" /></p>
+
+Specifies the hostnames that Astro is allowed to respond to in `dev` or `preview` modes. Can be passed a comma-separated list of hostnames or `true` to allow any hostname.
+
+Refer to the [Vite documentation](https://vite.dev/config/server-options.html#server-allowedhosts) for more information.
```

Comment:
```suggestion
Refer to [Vite's `allowHosts` feature](https://vite.dev/config/server-options.html#server-allowedhosts) for more information, including security implications of allowing hostnames.
```
Noting that at the Vite link (and, when researching allowed hosts in general), there tend to be warnings *not* to just enable `true` for security reasons. 

Do we similarly want guidance/warnings about this ourselves? A hint of something that exists in the Vite docs like above? Nothing?

Reply:
I leave the decision to you. In my initial design, I wanted to defer the documentation to `vite`, but if that's not the case, then I think we should add a warning